discovery-infra/test_infra/tools/static_ips.py

```diff
@@ -78,8 +70,8 @@ def _genetate_ips(mac_addresses, cidrs, address_offset):
         ipv6_conf = next(ipv6_gen)
         static_ips.append({
             'mac': mac,
-            'ip': ipv4_conf.ip, 'mask': ipv4_conf.mask, 'gateway': ipv4_conf.gateway, 'dns': ipv4_conf.dns,
-            'ip_v6': ipv6_conf.ip, 'mask_v6': ipv6_conf.mask, 'gateway_v6': ipv6_conf.gateway, 'dns_v6': ipv6_conf.dns
+            'ipv4_config': ipv4_conf,
+            'ipv6_config': ipv6_conf
```

Comment:
You don't need the ipv4_conf/ipv6_conf variables any more.
```
'ipv4_config': next(ipv4_gen)
'ipv6_config': next(ipv6_gen)
```

Reply:
Are the service changes already in? AFAIK it's a breaking API change.

Reply:
fixed

Reply:
and the service changes are in